javascript-forms/form-events.js

```diff
@@ -0,0 +1,24 @@
+function handleFocus(event) {
+  console.log('focus event was fired');
+  console.log(event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('the blur event was fired');
+  console.log(event.target.name);
+}
+
+function handleInput(event) {
+  console.log('value of name:', event.target.value);
+}
+
+var input = document.querySelector('input');
```

Comment:
instead of just selecting `input` you need to individually query them. there should be 3 inputs, and 9 event listeners